Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -434,13 +434,19 @@ void USpatialReceiver::ApplyComponentData(Worker_EntityId EntityId, Worker_Compo
 		TSet<UnrealObjectRef> UnresolvedRefs;
 
 		ComponentReader Reader(NetDriver, ObjectReferencesMap, UnresolvedRefs);
-		Reader.ApplyComponentData(Data, TargetObject, Channel);
+		Reader.ApplyComponentData(Data, TargetObject, Channel, /* bIsHandover */ false);
```

Comment:
I think if we're having to send is we're handover or not, maybe we should just make it an enum and do the same for all components?

Reply:
We could. I personally don't think it should be a priority though as it's going to add little value for the effort.